templates/default/gres.conf.erb

```diff
@@ -0,0 +1 @@
+NodeName=dummy-compute Name=gpu Type=tesla File=/dev/nvidia0
```

Comment:
Is this how we want the default node to appear? How does it effect sinfo?

Reply:
If instance does not have GPU, when we update `slurm_parallelcluster_nodes.conf` in `sqswatcher` we will not include GPU information for a node. As long as GPU information is not mentioned in `slurm_parallelcluster_nodes.conf`, slurm will not look at `gres.conf`. So having this default node here will not affect existing behavior.